api/db/migrations/20250917091232_rename-pix-teams-table.js

```diff
@@ -0,0 +1,28 @@
+const FORMER_TABLE_NAME = 'pix-teams';
+const NEW_TABLE_NAME = 'administration-teams';
```

Comment:
Je viens de me souvenir des nouvelles normes sur les noms de tables pour utiliser les underscores `_` :  https://github.com/1024pix/pix/blob/dev/docs/database.md

```suggestion
const NEW_TABLE_NAME = 'administration_teams';
```

Reply:
Merci @P-Jeremy !!! 🫶